versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -71,6 +71,9 @@ spec:
         - ms
         - micros
         - nanos
+      pattern:
+        description: Field value pattern
```

Comment:
Maybe say that it's a regular expression pattern?

Also consider adding an `example`, like https://github.com/elastic/package-spec/blob/33b54fb5403b6cc36449dcba1825b5ed7d02a3e8/versions/1/data_stream/manifest.spec.yml#L19-L20.

Reply:
Rephrased description and added one example.